src/components/look/look-list/style-tag-list/index.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+import { message } from 'antd';
+
+import { Button } from '@src/component/atoms';
+
+import { StyleTag } from '@src/types';
+import styleTagsData from '@src/data/style-tags.json';
+import { LookService } from '@src/services';
+
+type StyleTagListProps = {
+  styleTags: StyleTag[];
+  id: number;
+};
+
+function StyleTagList({ styleTags, id }: StyleTagListProps) {
+  const [tags, setTags] = useState<StyleTag[]>([...styleTags]);
+
+  const handleCheck = (tag: StyleTag) => {
+    tags.find((t) => t.id === tag.id)
+      ? setTags(tags.filter((t) => tag.id !== t.id))
+      : setTags((prevStayleTags) => [...prevStayleTags, tag]);
+  };
+
+  const handleSave = async () => {
+    await LookService.partialUpdate(id, { styleTags: tags });
```

Comment:
```suggestion
    await LookService.partialUpdate(id, { styleTagIds: selectedTagIds });
```

nested object를 update할땐 Foreign key를 업데이트해주세용

Reply:
앗 그럼 id가 Foreign key가 되어야한단 말씀이신가용? 잘 이해가 되지않았어요..ㅠㅠ!

Reply:
예를 들어 어떤 포스트의 작성자를 변경하고 싶으면,
partialUpdate할 때 user object 전체 대신 userId만 변경할 유저의 id로(Foreign key) 넘겨주면 됩니다!

Reply:
d3d4728

Reply:
params type이 ILookUpdateDTO= Partial<ILookInputDTO>로 변경해주기전에는 빨간줄이 떴어서 저렇게 변경했는데요
그래도 403 에러가 나옵니다 ㅠㅠ 어떤 부분이 잘못된 것인지 모르겠습니다...!ㅜ